packages/medusa/src/loaders/module-definitions/definitions.ts

```diff
@@ -4,7 +4,7 @@ export const MODULE_DEFINITIONS: ModuleDefinition[] = [
   {
     key: "stockLocationService",
     registrationName: "stockLocationService",
-    defaultPackage: false,
+    defaultPackage: "@medusajs/stock-location",
```

Comment:
I think this should still be `false` as long as we keep developing before release since we have opted for not having a dedicated feature flag. @srindom wdyt?